setup.py

```diff
@@ -10,10 +10,12 @@
     url="https://github.com/hcarvalhoalves/python-pmap",
     package_dir={'': 'src'},
     packages=find_packages('src'),
-    version="1.2.0",
+    version="1.3.0",
```

Comment:
Major bump?

Reply:
Since it's not a breaking change, I did a minor bump.